src/LodeRunner.API.Test/IntegrationTests/ExecutingTestRun/TestRunExecution.cs

```diff
@@ -256,22 +267,24 @@ private async Task<int> TryParseProcessOutputAndGetAPIListeningPort(List<string>
         private async Task<string> TryParseProcessOutputAndGetClientStatusId(List<string> outputList, int timeBetweenTriesMs = 500, int maxRetries = 10)
         {
             string clientStatusId = null;
-            for (int i = 1; i <= maxRetries; i++)
+            var taskSource = new CancellationTokenSource();
+
+            await Common.RunAndRetry(maxRetries, timeBetweenTriesMs, taskSource, async (int attemptCount) =>
             {
                 await Task.Delay(timeBetweenTriesMs).ConfigureAwait(false); // Log Interval is 5 secs.
 
```

Comment:
Same question here regarding the Task.Delay

Reply:
No need for it, will remove